src/filters/FiltersReducer.js

```diff
@@ -22,9 +22,6 @@ let initialState = {
 const filters = (state = initialState, action) => {
   switch (action.type) {
 
-  case types.DATA_CONFIG_REQUEST:
-    return {...state, loadingFilters: true};
```

Comment:
because it's totally awesome. it's less noisy than `Object.assign` and the redux docs talk about it http://redux.js.org/docs/recipes/UsingObjectSpreadOperator.html